sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Security/Ucfg/UcfgBuilder_Instructions.cs

```diff
@@ -491,6 +496,223 @@ public interface IBar<T>
             UcfgVerifier.VerifyInstructions(code, "Foobar");
         }
 
+        [TestMethod]
+        public void Invocations_MethodsWithDefaultParameters()
+        {
+            const string code = @"
+namespace Namespace
+{
+    public class Class1
+    {
+        public void Foo()
+        {
+            SendEmailAsync(""body"");                   // %0 := Namespace.Class1.SendEmailAsync(string, bool) [ this const ]
+            SendEmailAsync(""body"", isHtml: true);     // %1 := Namespace.Class1.SendEmailAsync(string, bool) [ this const const ]
+            SendEmailAsync(""body"", isHtml: false);    // %2 := Namespace.Class1.SendEmailAsync(string, bool) [ this const const ]
+        }
+
+        public System.Threading.Tasks.Task SendEmailAsync(string body, bool isHtml = false)
+        { return null; }
+    }
+}";
+            UcfgVerifier.VerifyInstructions(code, "Foo");
+        }
+
+        [TestMethod]
+        public void Invocations_UsedNamedParameters()
+        {
+            const string code = @"
+namespace Namespace
+{
+    public class Class1
+    {
+        public void Foo(string s)
+        {
+            SendEmailAsync(s, false);                  // %0 := Namespace.Class1.SendEmailAsync(string, bool) [ this s const ]
+            SendEmailAsync(body: s, isHtml: false);    // %1 := Namespace.Class1.SendEmailAsync(string, bool) [ this s const ]
+            SendEmailAsync(body: s);                   // %2 := Namespace.Class1.SendEmailAsync(string, bool) [ this s ]
+            SendEmailAsync(isHtml: false, body: s);    // %3 := Namespace.Class1.SendEmailAsync(string, bool) [ this const s ]

```

Comment:
Let's create a ticket to fix this... We are not passing the args in the right order.